project_management/online-collaboration-cafe.md

```diff
@@ -115,18 +115,39 @@ The _Turing Way_ online collaboration cafes will happen on the first and third W
-| 18 September 2019 | [https://youtu.be/ XUw5kpypeo8](https://youtu.be/XUw5kpypeo8) | [AreWeMeetingYet? 2019-09-18 19:00](https://arewemeetingyet.com/London/2019-09-18/19:00/TuringWay-CollaborationCafe) | 4am | 2am | 11:30pm | 8pm | 7pm | 2pm | 11am |
-| 19 September 2019 | [https://youtu.be/ 04vAi8TcBiA](https://youtu.be/04vAi8TcBiA) | [AreWeMeetingYet? 2019-09-19 19:00](https://arewemeetingyet.com/London/2019-09-19/19:00/TuringWay-CollaborationCafe) | 4am | 2am | 11:30pm | 8pm | 7pm | 2pm | 11am |
-| 2 October 2019 | [https://youtu.be/ HIIJvDZ8pzw](https://youtu.be/HIIJvDZ8pzw) | [AreWeMeetingYet? 2019-10-02 15:00](https://arewemeetingyet.com/London/2019-10-02/15:00/TuringWay-CollaborationCafe) | midnight | 10pm | 7:30pm | 4pm | 3pm | 10am | 7am |
-| 16 October 2019 | | [AreWeMeetingYet? 2019-10-16 19:00](https://arewemeetingyet.com/London/2019-10-16/19:00/TuringWay-CollaborationCafe) | 4am | 2am | 11:30pm | 8pm | 7pm | 2pm | 11am |
-| 30 October 2019 | | [AreWeMeetingYet? 2019-10-30 08:00](https://arewemeetingyet.com/London/2019-10-30/08:00/TuringWay-CollaborationCafe) | 6pm | 4pm | 1:30pm | 9am | 8am | 4am | 1am |
-| 6 November 2019 | | [AreWeMeetingYet? 2019-11-06 15:00](https://arewemeetingyet.com/London/2019-11-06/15:00/TuringWay-CollaborationCafe) | 1am | 11pm | 8:30pm | 4pm | 3pm | 10am | 7am |
-| 20 November 2019 | | [AreWeMeetingYet? 2019-11-20 19:00](https://arewemeetingyet.com/London/2019-11-20/19:00/TuringWay-CollaborationCafe) | 5am | 3am | 00:30am | 8pm | 7pm | 2pm | 11am |
-| 4 December 2019 | | [AreWeMeetingYet? 2019-12-04 15:00](https://arewemeetingyet.com/London/2019-12-04/15:00/TuringWay-CollaborationCafe) | 1am | 11pm | 8:30pm | 4pm | 3pm | 10am | 7am |
-| 18 December 2019 | | [AreWeMeetingYet? 2019-12-18 19:00](https://arewemeetingyet.com/London/2019-12-18/19:00/TuringWay-CollaborationCafe) | 5am | 3am | 00:30am | 8pm | 7pm | 2pm | 11am |
+***Video recordings from these calls will be posted on [our YouTube channel](https://www.youtube.com/channel/UCPDxZv5BMzAw0mPobCbMNuA).***
+
+#### Schedule for 2020
+
+ Date | Local Time | Brisbane | Shanghai | Kolkata | Paris | London | New York | Vancouver |
+| ---- | ---------- | --------- |-------- | ------- | ----- | ------ | -------- | --------- |
+| 15 January 2020 | [AreWeMeetingYet? 2020-01-15 19:00](https://arewemeetingyet.com/London/2020-01-15/19:00/TuringWay-CollaborationCafe) | 4am | 2am | 11:30pm | 8pm | 7pm | 2pm | 11am |
+| 5 February 2020 | [AreWeMeetingYet? 2020-02-05 15:00](https://arewemeetingyet.com/London/2020-02-05/15:00/TuringWay-CollaborationCafe) | midnight | 10pm | 7:30pm | 4pm | 3pm | 10am | 7am |
+| 19 February 2020 | [AreWeMeetingYet? 2020-02-19 19:00](https://arewemeetingyet.com/London/2020-02-19/19:00/TuringWay-CollaborationCafe) | 4am | 2am | 11:30pm | 8pm | 7pm | 2pm | 11am |
```

Comment:
@malvikasharan, could you check these times again? I think the time differences have changed now that we're out of summertime - Brisbane, at least, is currently 10 hours ahead rather than 9. I haven't checked all the others but we might hit the same problem there.